src/da65/handler.c

```diff
@@ -508,9 +546,24 @@ void OH_DirectIndirectLongX (const OpcDesc* D attribute ((unused)))
 
 
 
+static void impl_StackRelativeIndirectY (const char *sp, const OpcDesc* D attribute ((unused)))
```

Comment:
Good catch. The original impl_* function was much longer, but got shortened during development, so that I didn't recognize the redundancy in there.